src/components/molecules/UserProfileHeader.tsx

```diff
@@ -30,33 +35,92 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 interface IUserProfileHeader {
-  userDetails: IUserInfo;
+  userDetails: IAnywayUserDetails;
   isUpdateScreenOpen: boolean;
   handleLogout: () => any;
+  isAdmin: boolean;
 }
-const UserProfileHeader: React.FC<IUserProfileHeader> = ({ userDetails, isUpdateScreenOpen, handleLogout }) => {
+const UserProfileHeader: React.FC<IUserProfileHeader> = ({
+  userDetails,
+  isUpdateScreenOpen,
+  handleLogout,
+  isAdmin,
+}) => {
   const { t } = useTranslation();
-  const userData = userDetails.data;
+  const [loading, setLoading] = useState(false);
+  const store: RootStore = useStore();
+  const defaultFormInput: IFormInput = {
+    email: userDetails.data.email,
+    firstName: userDetails.data.firstName,
+    lastName: userDetails.data.lastName,
```

Comment:
Not a must, but can be nice:

```suggestion
  const { email, firstName, lastName} = userDetails.data
  const defaultFormInput: IFormInput = {
    email,
    firstName,
    lastName,
```